modal_proto/api.proto

```diff
@@ -855,10 +855,12 @@ message Image {
   // Note: field 13 is getting replaced field 16. The client now sends both fields,
   // but the server still only reads field 13.
   bool gpu = 13;
-  string build_function_def = 14;
   string context_mount_id = 15;
   GPUConfig gpu_config = 16;
   ImageRegistryConfig image_registry_config = 17;
+
+  string build_function_def = 14;
+  bytes build_function_globals = 18;
```

Comment:
Can we just serialize this into the previous field? Since I think we already do that client side with other arguments

Reply:
Was going to do that earlier but I thought it'd be nice to be able to "turn off" using `build_function_globals` in the image hash if it turned out to have issues. Also think we should separate out the other arguments at some point into a separate PB as we discussed